docker/centos/7/xpu_ispc_build/Dockerfile

```diff
@@ -13,6 +13,7 @@ ARG LLVM_VERSION=15.0
 ARG L0L_VER=1.13.5
 ARG VC_INTRINSICS_COMMIT_SHA="910db4801d4a029834606e3e42a8d60358e74fdf"
 ARG SPIRV_TRANSLATOR_COMMIT_SHA="e82ecc2bd7295604fcf1824e47c95fa6a09c6e63"
+ARG TBB=default
```

Comment:
Would be good to document here possible values and the reason we need two of them (when either of them should be used).